pkg/occlient/occlient.go

```diff
@@ -762,6 +764,10 @@ func (c *Client) FollowBuildLog(buildName string) error {
 	}
 	defer rd.Close()
 
+	// Set the colour of the stdout output..
+	color.Set(color.FgYellow)
```

Comment:
will this work on Windows. 
Accroding to docs you have to use 
```
fmt.Fprintf(color.Output, "Windows support: %s", color.GreenString("PASS"))
```

to print in color on windows

Reply:
I tried to find an alternative (coloured output by modifying stdout), but since we stream the data, I couldn't do it. 

I think we should just support Mac OS X and Linux for now.